clients/client-cognito-identity/runtimeConfig.browser.ts

```diff
@@ -1,4 +1,4 @@
-import { name, version } from "./package.json";
+import package from './package.json'; 
```

Comment:
Fix for build error:
```console
runtimeConfig.browser.ts(1,8): error TS1214: Identifier expected. 'package' is a reserved word in strict mode. Modules are automatically in strict mode.
```

```suggestion
import packageInfo from './package.json'; 
```